parts/kubernetesagentcustomdata.yml

```diff
@@ -13,6 +13,7 @@ runcmd:
 - systemctl restart docker
 - /usr/bin/curl -sSL --retry 12 --retry-delay 10 https://storage.googleapis.com/kubernetes-release/release/{{{kubectlVersion}}}/bin/linux/amd64/kubectl > /usr/local/bin/kubectl
 - chmod +x /usr/local/bin/kubectl
+- /bin/mkdir -p /etc/kubernetes/manifests
```

Comment:
Please use `mkdir`. If `/usr/bin` or `/bin` are never not-on-PATH, the system won't boot and we'll be so hosed it won't matter.